bucket/MPC-Video-Renderer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.6.7",
+    "description": "MPC Video Renderer is a free and open-source video renderer for DirectShow. ",
+    "homepage": "https://github.com/Aleksoid1978/VideoRenderer",
+    "license": "GPL-3.0",
+    "suggest": {
+        "Media Player Classic - Home Cinema fork": "extras/mpc-hc-fork"
+    },
+    "url": "https://github.com/Aleksoid1978/VideoRenderer/releases/download/0.6.7/MpcVideoRenderer-0.6.7.2013.zip",
+    "hash": "215D1F382A4A399052D7B3618C082BAC658DF8A03FF671471C028FC52DB5FCA9",
+    "pre_install": [
+        "if ($architecture -eq '64bit') { Start-Process -FilePath \"$dir\\Install_MPCVR_64.cmd\" -Verb RunAs }",
+        "elseif ($architecture -eq '32bit') { Start-Process -FilePath \"$dir\\Install_MPCVR_32.cmd\" -Verb RunAse }"
+    ],
+    "uninstaller": {
+        "script": [
+            "if ($architecture -eq '64bit') { Start-Process -FilePath \"$dir\\Uninstall_MPCVR_64.cmd\" -Verb RunAs }",
+            "elseif ($architecture -eq '32bit') { Start-Process -FilePath \"$dir\\Uninstall_MPCVR_32.cmd\" -Verb RunAse }"
+        ]
+    },

```

Comment:
```suggestion
```

Permission popup

Reply:
sorry did not get it what you mean by using architecture.bin 

and using note like this?
```
"notes": [
        "To install for '64bit' architecture, run '$dir\\Install_MPCVR_64.cmd' and for uninstall run '$dir\\Uninstall_MPCVR_64.cmd' as administrator, respectively.",
        "To install for '32bit' architecture, run '$dir\\Install_MPCVR_32.cmd' and for uninstall run '$dir\\Uninstall_MPCVR_32.cmd' as administrator, respectively.",
    ]
```

Reply:
![screenshot_230503_100247](https://user-images.githubusercontent.com/4353480/235887403-bea467de-5c2d-4858-b041-05c18f2bffb9.png)

I don't use mpc, but it does not look like it's necessary.

Reply:
its is necessary if you have a HDR Display like its automatically transfer HDR Metadata to Displays and trigger HDR mode on Display
- Transferring HDR10 data to the display.
i was using this Video Renderer for this purpose

Reply:
@HUMORCE hey what about this 
https://github.com/ScoopInstaller/Extras/pull/11172#discussion_r1183460981